infra/src/main/kotlin/fi/fta/geoviite/infra/map/AlignmentGeometry.kt

```diff
@@ -18,20 +26,46 @@ enum class MapAlignmentType {
     REFERENCE_LINE,
 }
 
-data class AlignmentHeader<T, U>(
-    val id: DomainId<T>,
-    val version: LayoutRowVersion<T>?,
-    val trackNumberId: DomainId<TrackLayoutTrackNumber>?,
+sealed class AlignmentHeader<T, U> {
```

Comment:
Erottelin nämä omikseen. Uudella mallilla näitä ei ollut mahdollista tyypittää samalla T:llä koska geometriapuoli ei ole layoutasset, mutta on noilla oikeastikin eri sisällöt että tämä olisi kannattanut tehdä muutenkin.